doc/command-line.md

```diff
@@ -2092,7 +2114,7 @@ The following `coltype`s are currently supported for picklists:
 * `gather` - use the CSV output of `sourmash gather` as a picklist
 * `prefetch` - use the CSV output of `sourmash prefetch` as a picklist
 * `search` - use the CSV output of `sourmash prefetch` as a picklist
-* `manifest` - use the CSV output of `sourmash sig manifest` as a picklist
+* `manifest` - use CSV manifests produced by `sig manfiest` as a picklist
```

Comment:
```suggestion
* `manifest` - use CSV manifests produced by `sig manifest` as a picklist
```

do we want to say `sig collect`/`sig check` here instead, if `sig manifest` is not the best way to produce standalone manifests?

Reply:
funny story, you can use `sig manifest` output as a picklist just fine 😆